core/templates/dev/head/components/question-directives/question-player/question-player.constants.ajs.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Constants for the question player directive.
+ */
+
+import { QuestionPlayerConstants } from
+  'components/question-directives/question-player/question-player.constants.ts';
+
+angular.module('oppia').constant(
```

Comment:
Shouldn't we be converting these and others to Angular constants?

Reply:
As you can see [here](https://www.competa.com/blog/migrate-angularjs-to-angular-using-upgrademodule-part-2/), the constants are to be declared as Angular as well as AngularJS to be available in the hybrid.